src/translators/timescale.py

```diff
@@ -234,6 +255,8 @@ def _db_value_to_ngsi(self, db_value: Any, ngsi_type: str) -> Any:
     # 2. Basic types (int, float, boolean and text). They also get converted
     # back to their corresponding Python types.
 
+    # TODO with the new pg8000 PGJsonb is removed...
+    # it simply replace with json dumps()
```

Comment:
hang on, isn't this function called from the sql translator? how come it doesn't bomb out?